docs/koa.asciidoc

```diff
@@ -0,0 +1,190 @@
+[[koa]]
+== Getting started with Koa
+
+Getting Elastic APM set up for your Koa app is easy,
+and there are various ways you can tweak it to fit to your needs.
```

Comment:
I won't continue to mark duplicate corrections. Just make sure you do a search to fix issues in all instances of duplicated text.